src/_00_IntroToArrays/RobotRacerNoRecipe.java

```diff
@@ -7,27 +7,49 @@
 import org.jointheleague.graphical.robot.Robot;
 
 public class RobotRacerNoRecipe {
-	static Robot[] robotList = new Robot[5];
+	private static int actualWinner;
+	private static int userBetWinner;
+	private static int usersNumOfRobots;
+	static Robot[] robotList;
 
 	public static void main(String[] args) {
 
-		Random randNum = new Random(100);
-		int randomNumber;
-		
-		String userAnswer = JOptionPane.showInputDialog("Place your bets! (Enter the number of robot you think will win the race)");
-		int numberOfWinner = Integer.parseInt(userAnswer) - 1;
-		
+		Random randNum = new Random();
+
+		String userResponse = JOptionPane.showInputDialog("How many robots do you want to race? (Between 1-9 robots, please.)");
+		try {
+			usersNumOfRobots = Integer.parseInt(userResponse);
+		} catch (NumberFormatException e) {
+			JOptionPane.showMessageDialog(null, "Letters are not acceptable values. Defaulting to 5 robots.");
+			usersNumOfRobots = 5;
+		}
+
+		while (usersNumOfRobots > 9 || usersNumOfRobots < 1) {
```

Comment:
Magic numbers! create fancy description names for these guys 😄

Reply:
👍